Makefile

```diff
@@ -16,9 +20,8 @@ export CGO_ENABLED=0
 
 # Environmental details
 BINDIR := rootfs/usr/local/bin
-VERSION := 0.0.1-$(shell date "+%Y%m%d%H%M%S")
-LDFLAGS := "-s -X main.version=${VERSION}"
-IMAGE := ${DEIS_REGISTRY}/deis/${SHORT_NAME}:${VERSION}
+LDFLAGS := "-s -X main.version=${BUILD_TAG}"
+IMAGE := ${DEIS_REGISTRY}/deis/${SHORT_NAME}:${BUILD_TAG}
```

Comment:
I think we should omit the `/` after `${DEIS_REGISTRY}` because it makes it impossible to push to docker hub (I just ran into this problem) cc/ @mboersma